sqlglot/expressions.py

```diff
@@ -3284,6 +3284,191 @@ class Update(Expression):
         "limit": False,
     }
 
+    def table(
+        self, expression: ExpOrStr, dialect: DialectType = None, copy: bool = True, **opts
+    ) -> Update:
+        """
+        Set the table to update.
+
+        Example:
+            >>> Update().table("my_table").set_("x = 1").sql()
+            'UPDATE my_table SET x = 1'
+
+        Args:
+            expression : the SQL code strings to parse.
+                If a `Table` instance is passed, this is used as-is.
+                If another `Expression` instance is passed, it will be wrapped in a `Table`.
+            dialect: the dialect used to parse the input expression.
+            copy: if `False`, modify this expression instance in-place.
+            opts: other options to use to parse the input expressions.
+
+        Returns:
+            The modified Update expression.
+        """
+        return _apply_builder(
+            expression=expression,
+            instance=self,
+            arg="this",
+            into=Table,
+            prefix=None,
+            dialect=dialect,
+            copy=copy,
+            **opts,
+        )
+
+    def set_(
+        self, *expressions: ExpOrStr, dialect: DialectType = None, copy: bool = True, **opts
+    ) -> Update:
```

Comment:
We probably want an "append" mode like we do for, e.g., `select`. Right now you always overwrite which isn't as flexible.

Reply:
Good idea, done